tests/stubs/edge/handlerstub/module.go

```diff
@@ -67,5 +67,8 @@ func (hs *HandlerStub) Start() {
 		klog.Errorf("New upstream controller failed with error: %v", err)
```

Comment:
we got no chance that `err` is not nil here, need to revisit how `NewUpstreamController` is defined.

Reply:
```
// Set sets the value of the named command-line flag.
func Set(name, value string) error {
	return CommandLine.Set(name, value)
}
```
pflag.Set would return err.

Now NewUpstreamController always return nil error, but handle it to check is fine for further optimization.